package.json

```diff
@@ -75,6 +75,7 @@
     "node-sass": "^6.0.0",
     "node-sass-glob-importer": "^5.3.2",
     "nodemon": "^2.0.4",
+    "prettier": "^2.4.1",
```

Comment:
It looks like the prettier config is not using WP standards - in themes, I believe this was resolved by using the Automattic version of the package: "prettier": "npm:wp-prettier@^2.2.1-beta-1"

Unless you know of another way to do this with prettierconfig?